.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to help us improve
```

Comment:
This reads like its for a new feature. Can you change the text to be something like "File a bug report".